modules/reference/pages/properties/broker-properties.adoc

```diff
@@ -135,6 +135,32 @@ TIP: For backward compatibility, `true` is the default. Redpanda recommends usin
 
 ---
 
+=== fips_mode
+
+Controls whether Redpanda starts in FIPS mode.  This property allows for three values: `disabled`, `enabled`, and `permissive`.  
+
+With `enabled`, Redpanda first verifies that the operating system is enabled for FIPS by checking `/proc/sys/crypto/fips_enabled`.  If the file does not exist or does not return `1`, Redpanda immediately exits.  
+
+With `permissive`, the same check is performed but a WARNING is logged and Redpanda continues to run.  After the check is complete, Redpanda loads the OpenSSL FIPS provider into the OpenSSL library. After this is complete, Redpanda is operating in FIPS mode, which means that the TLS cipher suites available to users are limited to TLSv1.2 and TLSv1.3, and of those, only the ones that use NIST-approved cryptographic methods.  
+
+For more information about FIPS refer to the Redpanda documentation.
```

Comment:
We shouldn't have self references to the documentation. This was extracted from the source code. 
```suggestion

```

Reply:
+1